python/ray/train/tests/test_tensorflow_predictor.py

```diff
@@ -69,7 +76,7 @@ def test_repr():
     assert pattern.match(representation)
 
 
-def create_checkpoint_preprocessor() -> Tuple[TensorflowCheckpoint, Preprocessor]:
+def create_checkpoint_preprocessor() -> Tuple["TensorflowCheckpoint", Preprocessor]:
```

Comment:
this is required so that `TensorflowCheckpoint` as a type does not scream at runtime